posts/2021-03-24-microprofile40-open-liberty-21003.adoc

```diff
@@ -0,0 +1,442 @@
+---
+layout: post
+title: "MicroProfile 4.0 support in Open Liberty 21.0.0.3"
+categories: blog
+author_picture: https://avatars.githubusercontent.com/u/10999780?s=460&u=1a66e468c70e84f54bf5a75c27881cc6133ccc20&v=4
+author_github: https://github.com/Emily-Jiang
+seo-title: MicroProfile 4.0 support in Open Liberty 21.0.0.3 - OpenLiberty.io
+seo-description: This post demonstrates how to use MicroProfile 4.0 in Open Liberty 21.0.0.3.
```

Comment:
The latest release of MicroProfile 4.0 is the first release under the newly-formed MicroProfile Working Group, delivering alignment with Jakarta EE 8 as well as new and exciting updates to many of the core MicroProfile APIs: Config, Fault Tolerance, Health, JWT RBAC, Metrics, Open API, Open Tracing, Rest Client. This post dives into these updates in further detail and demonstrates how you can utilize MicroProfile 4.0 in Open Liberty 21.0.0.3.

I have extended and updated this description to hopefully increase our SEO.